File: 03_OO/Greet.c
#include <stdio.h>
#include <string.h>

typedef struct {
    char name[50];
    int age;
} Person;

void greet(Person p) {
    printf("私は%sです。%d歳です。\n", p.name, p.age);
}

int main (void) {
    Person bob;
    strcpy(bob.name, "Bob");
    bob.age = 19;

    greet(bob);

    return 0;
}

File: 03_OO/Greet.java
class Person {
    String name;
    int age;

    void greet() {
        System.out.println("私は" + name + "です。" + age + "歳です。");
    }
}

public class Greet {
    public static void main(String[] args) {
        Person bob = new Person();
        bob.name = "Bob";
        bob.age = 19;

        bob.greet();
    }
}

